site/gatsby-site/src/templates/cite.js

```diff
@@ -141,6 +147,59 @@ function CitePage(props) {
     );
   }, [user]);
 
+  const [subscribeToNewReportsMutation, { loading: subscribing }] =
+    useMutation(UPSERT_SUBSCRIPTION);
+
+  const subscribeToNewReports = async () => {
+    if (isRole('subscriber')) {
+      try {
+        await subscribeToNewReportsMutation({
+          variables: {
+            query: {
+              type: 'incident',
+              userId: { userId: user.id },
```

Comment:
I think this enables anyone to subscribe other users?

I wonder if there is something like subscribe "whoever is sending this request" to prevent that from happening.